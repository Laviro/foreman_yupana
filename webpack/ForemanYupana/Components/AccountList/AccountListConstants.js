export const ACCOUNTLIST_CHANGE_BOOL = 'ACCOUNTLIST_CHANGE_BOOL';
