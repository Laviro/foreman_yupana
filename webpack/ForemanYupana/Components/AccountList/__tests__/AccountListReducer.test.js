import { testReducerSnapshotWithFixtures } from 'react-redux-test-utils';

import { ACCOUNTLIST_CHANGE_BOOL } from '../AccountListConstants';
import reducer from '../AccountListReducer';

const fixtures = {
  'should return the initial state': {},
  'should handle ACCOUNTLIST_CHANGE_BOOL': {
    action: {
      type: ACCOUNTLIST_CHANGE_BOOL,
      payload: {
        bool: true,
      },
    },
  },
};

describe('AccountList reducer', () =>
  testReducerSnapshotWithFixtures(reducer, fixtures));
