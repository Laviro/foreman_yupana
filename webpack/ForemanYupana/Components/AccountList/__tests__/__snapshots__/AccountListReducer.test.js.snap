// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountList reducer should handle ACCOUNTLIST_CHANGE_BOOL 1`] = `
Object {
  "bool": true,
}
`;

exports[`AccountList reducer should return the initial state 1`] = `
Object {
  "bool": false,
}
`;
